content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -219,11 +219,23 @@ v1.13.x, etcd v2 data MUST by migrated to the v3 storage backend, and
 kube-apiserver invocations changed to use `--storage-backend=etcd3`.
 
 The process for migrating from etcd2 to etcd3 is highly dependent on how the
-etcd cluster was deployed and configured, as well as how the Kubernetes 
+etcd cluster was deployed and configured, as well as how the Kubernetes
 cluster was deployed and configured. We recommend that you consult your cluster
 provider's documentation to see if there is a predefined solution.
 
 If your cluster was created via `kube-up.sh` and is still using etcd2 as its
 storage backend, please consult the [Kubernetes v1.12 etcd cluster upgrade docs](https://v1-12.docs.kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#upgrading-and-rolling-back-etcd-clusters)
 
+## Known issues
+
+etcd servers may fail or disconnect briefly from kube-apiserver. Then, etcd client (embedded in kube-apiserver) should never deadlock being stuck with a downed node, and instead failover to other etcd nodes. etcd v3 client released in v3.3.13 (or before) had a critical bug that the client balancer failover does not properly work against secure endpoints. This affects kube-apiserver HA deployments.
```

Comment:
hello @gyuho 
- Known Issue: Is there more than one issue, or the same issue that affects the `kube-apiserver` because of the `etcd` client?
- You may want to begin the problem description with the etcd bug description. Suggested edits:
The etcd v3 client, released in etcd v3.3.13 or earlier, has a critical bug (link to the bug#?) which affects the `kube-apiserver` and HA deployments. The etcd client balancer failover does not properly work against secure endpoints. As a result, etcd servers may fail or disconnect briefly from the `kube-apiserver`.